<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vyrazu66-ThinkCentre-M70t" failures="2" tests="23" name="Different testng Features Test" time="17.799" errors="0" timestamp="27 Jul 2023 08:07:00 GMT">
  <testcase classname="Testng.ExceptionTimeOutTest" name="test1" time="0.006"/>
  <testcase classname="Testng.ExceptionTimeOutTest" name="infiniteloopTest" time="0.006">
    <failure type="org.testng.TestException" message="
Expected exception java.lang.NumberFormatException but got org.testng.TestException: 
Expected exception java.lang.NumberFormatException but got org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.infiniteloopTest() didn&amp;apos;t finish within the time-out 2">
      <![CDATA[org.testng.TestException: 
	... 16 more
	at java.base@17.0.5/sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:223)
	at java.base@17.0.5/sun.nio.cs.StreamEncoder.implFlushBuffer(StreamEncoder.java:313)
	at java.base@17.0.5/sun.nio.cs.StreamEncoder.flushBuffer(StreamEncoder.java:111)
	at java.base@17.0.5/java.io.OutputStreamWriter.flushBuffer(OutputStreamWriter.java:178)
	at java.base@17.0.5/java.io.PrintStream.writeln(PrintStream.java:723)
	at java.base@17.0.5/java.io.PrintStream.println(PrintStream.java:938)
	at app//Testng.ExceptionTimeOutTest.infiniteloopTest(ExceptionTimeOutTest.java:14)
	at java.base@17.0.5/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base@17.0.5/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.5/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.5/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.5/java.lang.Thread.run(Thread.java:833)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- infiniteloopTest -->
  <testcase classname="Testng.GoogletitleTest" name="test_title" time="0.007">
    <failure type="java.lang.AssertionError" message="title is not matched expected [Google789] but found [Google]">
      <![CDATA[java.lang.AssertionError: title is not matched expected [Google789] but found [Google]
	at Testng.GoogletitleTest.test_title(GoogletitleTest.java:34)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test_title -->
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.0"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.0"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.0"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="Testng.InvocationCountTest" name="sum" time="0.0"/>
  <testcase classname="Testng.TestNgFeatures" name="loginTest" time="0.0"/>
  <testcase classname="Testng.FacebookTest" name="test_title" time="0.008"/>
  <testcase classname="Testng.FacebookTest" name="googleLogotest" time="0.038"/>
  <testcase classname="Testng.FacebookTest" name="Linktest" time="0.034"/>
  <testcase classname="Testng.FacebookTest" name="test4" time="0.001"/>
  <testcase classname="Testng.FacebookTest" name="test5" time="0.001"/>
  <testcase classname="Testng.FacebookTest" name="test6" time="0.001"/>
  <testcase classname="Testng.TestNgFeatures" name="HomePageTest" time="0.001"/>
  <testcase classname="Testng.TestNgFeatures" name="RegistationPage" time="0.001"/>
  <testcase classname="Testng.TestNgFeatures" name="searchpageTest" time="0.0"/>
</testsuite> <!-- Different testng Features Test -->
